HouseholdManager/Views/Room/Details.cshtml

```diff
@@ -1,38 +1,46 @@
-﻿@model HouseholdManager.Models.Room
+﻿@model HouseholdManager.Models.ViewModels.RoomDetailViewModel
 @using HouseholdManager.Data.API
 
 @{
-    ViewData["Title"] = "Room Details";
+    ViewData["Title"] =  @Model.RoomName + " Details";
     var req = new IconRequestor();
-    string iconMouseOverText = await req.GetMouseoverTextForIcon(ViewData.Model.Icon,
-                                                                 (IEnumerable<Icon>)ViewBag.Icons);
+    //Below is no longer needed with the ViewModel in place
+    //string iconMouseOverText = await req.GetMouseoverTextForIcon(ViewData.Model.Icon,
+    //                                                             (IEnumerable<Icon>)ViewBag.Icons);
```

Comment:
This API call is actually still doing something.  We don't store mouseover text for the icon in the database, so we need to get it from the API here.

I know, having an <Icon> type and an "Icon" property that isn't the <Icon> type isn't ideal, and I'd definitely like to rework that if given the time.

```suggestion
    string iconMouseOverText = await req.GetMouseoverTextForIcon(Model.RoomIcon,(IEnumerable<Icon>)ViewBag.Icons);
```